client/components/tinymce/plugins/wpeditimage/plugin.js

```diff
@@ -68,6 +68,7 @@ function wpEditImage( editor ) {
 			'wp_img_alignnone',
 			'wpcom_img_size_decrease',
 			'wpcom_img_size_increase',
+			'wp_img_edit', // See plugins/media
```

Comment:
I'd move this down 2 lines so the button appears next to the `remove` / `delete` button